exasol/bucketfs/_path.py

```diff
@@ -41,6 +50,12 @@ def as_uri(self) -> str:
         Represent the path as a file URI. Can be used to reconstruct the location/path.
         """
 
+    def as_udf_path(self) -> str:
```

Comment:
should we maybe return a PosixPath

Reply:
Again, not a big issue, but the PathLike returns either strings or PathLikes elsewhere. We can't return a PathLike because the backend should be different. So, I think a string is the most appropriate.